x-pack/test/apm_api_integration/tests/service_overview/dependencies/index.ts

```diff
@@ -441,38 +444,33 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         const impactValues = sortBy(
           response.body.serviceDependencies.map((item) => ({
             name: item.name,
-            impact: item.impact,
-            latency: item.latency.value,
```

Comment:
Was latency not used here?

Reply:
Then latency values are already being checked in another test https://github.com/elastic/kibana/blob/a1acc7edebc5b2e440b6d3cd9270ccf3f2bc75a1/x-pack/test/apm_api_integration/tests/service_overview/dependencies/index.ts#L371, that's why I removed it from here and left only the impact, which is what this test must validate.